pyro/ops/einsum/paths.py

```diff
@@ -56,7 +56,7 @@ def _get_candidate(output, sizes, remaining, footprints, dim_ref_counts, k1, k2)
     two = k1 & k2
     one = either - two
     k12 = (either & output) | (two & dim_ref_counts[3]) | (one & dim_ref_counts[2])
-    cost = 2 * _footprint(k12, sizes) - footprints[k1] - footprints[k2]
```

Comment:
This brings makes our objective function match the upstream `eager` objective exactly. We will be deleting this entire file before 0.3 release.

Reply:
Is the `opt_einsum` scheduled to release soon?

Reply:
Yes, @dgasmith [offered](https://github.com/dgasmith/opt_einsum/pull/74#issuecomment-441390408) to cut a minor release so we could get recent `opt_einsum` improvements into Pyro 0.3.